### ISSUE TITLE
Tusky 21 doesn't show emoji

### ISSUE THREAD

**Issue by alan-ford** [tuskyapp/Tusky#3638]
Tusky does not show emoji as opposed to other clients such as Fedilab.

* * * *
- Tusky Version: 21.0
- Android Version: 7.1.2
- Android Device: Amazon Fire
- Mastodon instance (if applicable): livellosegreto.it

![BUG](https://github.com/tuskyapp/Tusky/assets/44990009/4cf0a992-e521-47d3-8a38-7fa9d7fb1122)

**Comment by connyduck** [comment]
Do you have a link to these posts? Is it all emojis or only some? Does it change if you use a different emoji set (Preferences -> Emoji style)?

**Comment by alan-ford** [comment]
@connyduck I checked now, this problem occurs only with Noto Emoji. It does not show any emoji in any post.

**Comment by connyduck** [comment]
cc @C1710

**Comment by nikclayton** [comment]
If I've understood this properly, this can be closed, yes? The underlying problem is that the particular emoji set in use didn't have entries for those characters, and there's nothing Tusky can do?

**Comment by C1710** [comment]
> > If I've understood this properly, this can be closed, yes? The underlying problem is that the particular emoji set in use didn't have entries for those characters, and there's nothing Tusky can do?
> 
> We need to update the library once a fix is available, but since we get automatic PRs it can probably closed, yes.
> 
> @C1710 are you aware of this problem?

I actually forgot it. But I don't know yet where the problem could be. The font is the same as it's used by the original EmojiCompat library, so it *should* work. The only issue I found was that the current version of that font (which Tusky doesn't use) is incompatible with the underlying version of EmojiCompat that we use right now.

I'm planning to release a new version of my library in the next few weeks or so (i.e., when I finally figured out how to do the UI testing/have time for that). But since I don't know where the underlying problem is, I don't know if it'll be fixed.
